From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Fri, 5 Jun 2026 17:15:19 +0800
Subject: tg-bootstrap

This dependency is missing, but would pose problem on Darwin
---
 sysdeps/mach/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sysdeps/mach/Makefile b/sysdeps/mach/Makefile
index 5607f66..bbf5211 100644
--- a/sysdeps/mach/Makefile
+++ b/sysdeps/mach/Makefile
@@ -35,7 +35,7 @@ endif
 # because it's different in Darwin and the conditional crap is
 # too much trouble.  This should suffice for getting the mach/Makefile
 # rules invoked when they need to be.
-mach-before-compile := $(mach-objpfx)mach-shortcuts.h \
+mach-before-compile := $(mach-objpfx)mach-shortcuts.h $(mach-objpfx)mach/mach_interface.h \
 		       $(patsubst %,$(mach-objpfx)mach/mach_%.h,\
 				  port host)
 
